zulipterminal/ui_tools/boxes.py

```diff
@@ -61,8 +61,11 @@ def private_box_view(self, button: Any=None, email: str='',
         ]
         self.focus_position = 1
 
-    def stream_box_view(self, caption: str='', title: str='') -> None:
+    def stream_box_view(self, stream_id: int, caption: str='', title: str='',
+                        ) -> None:
         self.set_editor_mode()
+        self.recipient_user_ids = self.model.get_subscribers_in_stream(
```

Comment:
You seem to have lost this function in this iteration of the PR.

Worth adding a test?

Reply:
Writebox doesn't have any tests at all (except for typeahead) so I'll just add a test for this line and open an issue for the rest.

Reply:
Opened #739